src/github.com/getlantern/flashlight/proxiedsites/proxiedsites.go

```diff
@@ -52,6 +56,24 @@ func Configure(cfg *proxiedsites.Config, proxyAddr string) {
 	startMutex.Unlock()
 }
 
+func updateDetour(delta *proxiedsites.Delta) {
+	// TODO: subscribe changes of geolookup and set country accordingly
+	// safe to hardcode here as IR has all detection rules
+	detour.SetCountry("IR")
```

Comment:
Also, since there's just one detector right now, it might be easiest just to get rid of the country thing completely and only revisit this if and when we find the need to have different detection algorithms.